website/docs/r/batch_job_definition.html.markdown

```diff
@@ -61,6 +130,7 @@ The following arguments are supported:
 * `container_properties` - (Optional) A valid [container properties](http://docs.aws.amazon.com/batch/latest/APIReference/API_RegisterJobDefinition.html)
     provided as a single valid JSON document. This parameter is required if the `type` parameter is `container`.
 * `parameters` - (Optional) Specifies the parameter substitution placeholders to set in the job definition.
+* `platform_capability` - (Optional) The platform capabilities required by the job definition. If no value is specified, it defaults to `EC2`. Jobs run on Fargate resources specify `FARGATE`.
```

Comment:
according the the basic test check there is no value if one is not specified:

`resource.TestCheckResourceAttr(resourceName, "platform_capability.#", "0"),`
how does this conform with the above docs?

Reply:
It's maybe unclear but the `platformCapabilities` is optional and if not specified defaults to `EC2`, that means job will be only able to run on an EC2 compute environment. This idea of the test check is to be sure that this do not introduce a new behaviour and modify existing job definitions.

See AWS documentation: https://docs.aws.amazon.com/batch/latest/userguide/job_definition_parameters.html#job-definition-parameters-platform-capabilities